### PR TITLE
chore: update table examples

### PR DESCRIPTION
**Pull request by andrewseguin**
Updates the table examples to minimize styling and fix some small issues since moving to native table

### CODE REVIEW

File: src/material-examples/cdk-table-basic/cdk-table-basic-example.ts [hunk 1]
```diff
@@ -11,61 +11,29 @@ import {BehaviorSubject, Observable} from 'rxjs';
...
-
-    return {
-      id: (this.data.length + 1).toString(),
-      name: name,
-      progress: Math.round(Math.random() * 100).toString(),
-      color: COLORS[Math.round(Math.random() * (COLORS.length - 1))]
-    };
-  }
-}
+const ELEMENT_DATA: Element[] = [
```
Comment (jelbourn) [inline src/material-examples/cdk-table-basic/cdk-table-basic-example.ts]: `Element` is already a thing, probably don't overload that.

Comment (andrewseguin) [reply src/material-examples/cdk-table-basic/cdk-table-basic-example.ts]: Yup agreed - changed to `PeriodicElement`

**Comment by andrewseguin** [comment]
Changed Element to `PeriodicElement`; rebased and ready for review

**Comment by jelbourn** [comment]
@andrewseguin looks like there's a real failure on CI

**Pull request merged**